src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -461,19 +463,40 @@ worker_apply_sequence_command(PG_FUNCTION_ARGS)
 						" SEQUENCE command string")));
 	}
 
+	/*
```

Comment:
can we get this part to the master branch even though it is not that relevant on the master branch?

I think we can still simulate on the master branch with an explicit sequence on the worker?

Reply:
As we discussed offline, we decided to merge it with this PR.